code/go/0chain.net/smartcontract/storagesc/validator.go

```diff
@@ -41,6 +41,13 @@ func (sc *StorageSmartContract) addValidator(t *transaction.Transaction, input [
 			return "", common.NewError("add_validator_failed",
 				"Failed to get validator."+err.Error())
 		}
+
+		_, err = sc.getBlobber(newValidator.ID, balances)
+		if err != nil {
+			return "", common.NewError("add_validator_failed",
+				"failed to fetch blobber: "+err.Error())
```

Comment:
Maybe change the message. The error is that the new validator id does not match a registered blobber.

Reply:
done